src/syntax/FStar.Syntax.Util.fst

```diff
@@ -2497,3 +2497,53 @@ let flatten_refinement t =
     | _ -> t
   in
   aux t false
+
+let contains_strictly_positive_attribute (attrs:list attribute)
```

Comment:
Minor nit: Syntax.Util already has `has_attribute` which could be used here and in `contains _unused_attribute`.

Reply:
fixed